py/torch_tensorrt/dynamo/lowering/_decomposition_groups.py

```diff
@@ -175,6 +175,7 @@
     aten.linalg_vector_norm,
     aten.full,
     aten.repeat,
+    aten.var_mean,
```

Comment:
@gs-olive you mentioned the `rsub`. I will add that.
Regarding `sqrt` and `rsqrt`, sqrt is not of lowering type so I do not think it should be included here.
For `rsqrt`, should I include it here? Since `rsqrt` is already present in ``py/torch_tensorrt/dynamo/lowering/_decompositions.py`` and that would take precedence over the enabled ones, right? And we would not need to add it to the disabled ops then.

Reply:
That is true, I agree with your points. I think the `rsub` would be the main one to change; we can leave the others out, and the `rsqrt` implementation in `_decompositions.py` will take precedence, yes.